conda_forge_tick/cli.py

```diff
@@ -43,98 +43,120 @@ def deploy(args):
 
     # TODO: have function construct this
     BUILD_URL = os.environ.get(BUILD_URL_KEY, "")
-    cmds = ["git pull -s recursive -X theirs"]
-    cmds += [
-        "git add " + v
-        for v in [
-            "pr_json/*",
-            "status/*",
-            "node_attrs/*",
-            "audits/*",
-            "audits/grayskull/*",
-            "audits/depfinder/*",
-            "versions/*",
-            "profiler/*",
-            "mappings/*",
-            "mappings/pypi/*",
-        ]
-    ]
-    cmds += [f'git commit -am "Update Graph {BUILD_URL}"']
-
-    for cmd in cmds:
+
+    # pull changes, add ours, make a commit
+    try:
+        _run_git_cmd("git pull -s recursive -X theirs")
+    except Exception as e:
+        print(e)
+
+    files_to_add = (
+        # untracked
+        subprocess.run(
+            "git ls-files -o --exclude-standard",
+            shell=True,
+            capture_output=True,
+        ).stdout.decode("utf-8").splitlines()
+        # changed
+        + subprocess.run(
+            "git diff --name-only",
+            shell=True,
+            capture_output=True,
+        ).stdout.decode("utf-8").splitlines()
+        # modified and staged but not deleted
+        + subprocess.run(
+            "git diff --name-only --cached --diff-filter=d",
+            shell=True,
+            capture_output=True,
+        ).stdout.decode("utf-8").splitlines()
+    )
+    n_added = 0
+    for file in files_to_add:
+        if file is not None and len(file) > 0 and os.path.exists(file):
```

Comment:
would `bool(file)` suffice to replace the first two clauses

Reply:
It might but I'd be a lot less clear with a marginal performance gain.

Reply:
I simplified a bit here.